docs/getting_started/install_ez_enterprise.md

```diff
@@ -76,18 +70,33 @@ They are available via [updates.ez.no](https://updates.ez.no).
         They will become active again if the agreement is renewed, but this process may take up to 24 hours.
         _(If the agreement is renewed before the expiry date, there will be no disruption of service.)_
 
+    ## Create project
+
+    In order to install a new project using `composer create-project` to get latest version of eZ Enterprise,
+    you'll need to tell Composer which token to use before the project folder is created in the first place.
+    This can be done in the following way:
+
+    ``` bash
+    COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer create-project --keep-vcs --repository=https://updates.ez.no/bul/ ezsystems/ezplatform-ee my-new-ee-project
+    ```
+
     !!! tip "Usage of authentication token with `composer create-project`"
 
         If you have several projects set up on your machine,
         they should all use different tokens set in `auth.json` file in project directory.
 
-        But in order to install a new project using `composer create-project` to get latest version of eZ Enterprise,
-        you'll need to tell Composer which token to use before the project folder is created in the first place.
-        This can be done in the following way:
+    !!! note "Moving from trial"
 
-        ``` bash
-        COMPOSER_AUTH='{"http-basic":{"updates.ez.no":{"username":"<installation-key>","password":"<token-password>"}}}' composer create-project --repository=https://updates.ez.no/bul/ ezsystems/ezplatform-ee my-new-ee-project
-        ```
+        If you started with a trial installation, you should adjust the channel(s) you use in order to get software under [BUL license instead of a TTL license](https://ez.no/About-our-Software/Licenses-and-agreements/).
```

Comment:
```suggestion
        If you started with a trial installation, you should adjust the channel(s) you use in order to get software under [BUL license instead of a TTL license](https://ez.no/About-our-Software/Licenses-and-agreements/). See instructions below in "Edit composer.json" on how.`
```